### PR TITLE
Fixes mail delivery for Rails-3 apps

### PR DESCRIPTION
**Pull request by HashNuke**
Sets up SMTP for rails-3 apps instead of defining an ActionMailer delivery method.

### CODE REVIEW

**Comment by sgrove** [comment]
Does it still work with rails 2 apps?

File: lib/bushido.rb [hunk 1]
```diff
@@ -36,8 +36,13 @@ module Bushido #:nodoc:
     require "#{base_dir}/app/controllers/bushido/data_controller"
     require "#{base_dir}/app/controllers/bushido/mail_controller"
     require "#{base_dir}/app/controllers/bushido/envs_controller"
+    require "bushido/action_mailer"
```
Comment (HashNuke) [inline lib/bushido.rb:39]: this part to be specific

**Comment by sgrove** [comment]
So that means you've tested it?

**Comment by kevzettler** [comment]
:+1:

**Pull request closed without merging**